modules/administration/pages/removing-channels.adoc

```diff
@@ -117,25 +110,33 @@ spacewalk-remove-channel -a sles12-sp5-pool-x86_64
 //----
 
 
-==== Remove Packages from the Database via the {webui}
 
-.Procedure: Remove Packages from the Database via the {webui}
-. After removing the channel, you need to delete the packages from the database via the {webui}.
-. In the {productname} {webui} navigate to menu:SUSEManager[Software > Manage > Packages].
+== Remove Packages from the Database and the Filesystem
+
+To finally gain space remove the packages from the database and filesystem.
+
+
+
+=== Remove Packages from the Database Using {webui}
+
+.Procedure: Removing Packages from the Database Using {webui}
+. After removing the channel, you need to delete the packages from the database using the {webui}.
+. In the {productname} {webui}, navigate to menu:SUSEManager[Software > Manage > Packages].
 . Select [literal]``Package in no channel``, and delete the packages.
 
 
 
-==== Clean the Packages from the Filesystem
+===  Remove the Packages from the Filesystem
 
-.Procedure: Clean the Packages from the Filesystem
-. The following command checks for anything that is still showing in the filesystem, but not in the SUMA database and then removes it.
+.Procedure: Removing the Packages from the Filesystem
+. The following command checks for anything that still exists in the filesystem, but not in the {productname} database, and removes it.
+  On the command line of the container host, run the following command:
 +
 ----
-# spacewalk-data-fsck -r -S -C -O
+mgrctl exec -ti -- spacewalk-data-fsck -r -S -C -O
```

Comment:
spacewalk-data-fsck is a debugging and troubleshooting tool. It helps after fatal crashes when inconsistencies are detected. You do not need it to remove packages from disk after using of spacewalk-remove-channel.

I would move this into the troubleshooting section.

Reply:
ok.  I created a separate issue for further tracking: https://github.com/uyuni-project/uyuni-docs/issues/3632